pkgs/desktops/gnome/misc/gnome-flashback/default.nix

```diff
@@ -180,10 +180,13 @@ let
             dontWrapGApps = true; # We want to do the wrapping ourselves.
 
             # gnome-flashback and gnome-panel need to be added to XDG_DATA_DIRS so that their .desktop files can be found by gnome-session.
+            # We need to pass the --builtin flag so that gnome-session invokes gnome-session-binary instead of systemd.
+            # If systemd is used, it doesn't use the environment we set up here and so it can't find the .desktop files.
             preFixup = ''
               makeWrapper ${gnome-session}/bin/gnome-session $out \
-                --add-flags "--session=gnome-flashback-${wmName}" \
+                --add-flags "--session=gnome-flashback-${wmName} --builtin" \
                 --set-default XDG_CURRENT_DESKTOP 'GNOME-Flashback:GNOME' \
+                --prefix PATH : '${lib.makeSearchPath "bin" [ ibus ]}' \
```

Comment:
Is this just done to silence a warning that will disappear when user enables ibus in their configuration?

```
i18n.inputMethod.enabled = "ibus";
i18n.inputMethod.ibus.engines = with pkgs.ibus-engines; [ mozc ];
```